### DISCUSSION TITLE
请教：关于”实验性：绕过中国大陆IP“功能，其中的IP定义在哪里或者是从哪里获取的？

### DISCUSSION THREAD

**Discussion by SV-jiang-guangming** [vernesong/OpenClash#3284] [Q&A]
如题，"实验性：绕过中国大陆IP"功能的IP判断范围是否和”大陆白名单订阅“中的数据有关联？或者开启此功能后，如何新增一些IP地址或域名同样不经过openclash内核，
在访问控制中黑名单模式”本地IPv4绕过地址“新增域名”teamviewer.com“和”*.teamviewer.com“应用配置后该地址依然进入了内核；
在访问控制中黑名单模式”不走代理的局域网设备IP“和”Mac“功能中添加IP和MAC是可以实现不进入内核功能的；

如上描述，不知是”本地IPv4绕过地址“功能中只允许添加IP地址/掩码格式，还是域名格式问题，或者此功能理解不对。
请大家帮忙解惑[抱拳]

**Comment by Vegas7** [comment]
同样问题，在本地IPv4绕过地址中加入域名，只有443端口的域名可以生效，而80端口的仍然会经过内核。

**Comment by refined-fish** [comment]
是的，如你所想，关联文件就是大陆白名单订阅
